### ISSUE TITLE
Flex Align Icons

### ISSUE THREAD

**Issue by darren-outdev** [Templarian/MaterialDesign#5563]
## Usage

Much like the LEFT, CENTER, and JUSTIFY text aligns, this would be for alignment of a [Flexbox](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox) (to position its children). Suggestions for icons names would be:

flex-left
flex-right
flex-center
flex-none

## Examples

Include any example images so we know what the icon should look like.

![image](https://user-images.githubusercontent.com/71891985/95054079-9b75b680-06f1-11eb-9f12-ed031c252e03.png)

**Comment by Templarian** [comment]
@darren-outdev Updated your issue with some other names.

**Comment by zgover** [comment]
Proposal.

![image](https://user-images.githubusercontent.com/9044773/95369348-f1d32880-089c-11eb-90da-1cd554896c94.png)
[flex-align-baseline.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342831/flex-align-baseline.svg.zip)

![image](https://user-images.githubusercontent.com/9044773/95369395-fe578100-089c-11eb-8a3e-f83aa519e174.png)
[flex-align-center.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342833/flex-align-center.svg.zip)

![image](https://user-images.githubusercontent.com/9044773/95369431-0a434300-089d-11eb-9021-49fb2e7520b3.png)
[flex-align-end.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342835/flex-align-end.svg.zip)

![image](https://user-images.githubusercontent.com/9044773/95369476-17603200-089d-11eb-91ee-9f7c0d3547dd.png)
[flex-align-start.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342841/flex-align-start.svg.zip)

![image](https://user-images.githubusercontent.com/9044773/95369521-247d2100-089d-11eb-850f-1de12fce9983.png)
[flex-align-stretch.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342843/flex-align-stretch.svg.zip)

![image](https://user-images.githubusercontent.com/9044773/95369564-319a1000-089d-11eb-8194-1b4f1ad5a523.png)
[flex-justify-center.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342850/flex-justify-center.svg.zip)

![image](https://user-images.githubusercontent.com/9044773/95369600-3fe82c00-089d-11eb-84a2-42339e4fd78d.png)
[flex-justify-end.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342852/flex-justify-end.svg.zip)

![image](https://user-images.githubusercontent.com/9044773/95369632-4d9db180-089d-11eb-8cd6-cd9c34f15dcb.png)
[flex-justify-space-arround.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342857/flex-justify-space-arround.svg.zip)

![image](https://user-images.githubusercontent.com/9044773/95369682-5c846400-089d-11eb-92ec-a025ed22a3c9.png)
[flex-justify-space-between.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342859/flex-justify-space-between.svg.zip)

![image](https://user-images.githubusercontent.com/9044773/95369716-66a66280-089d-11eb-9c37-06413a16069b.png)
[flex-justify-start.svg.zip](https://github.com/Templarian/MaterialDesign/files/5342861/flex-justify-start.svg.zip)

**Comment by Xenomorph99** [comment]
If not sure if others feel this way or not, but personally I find it hard to understand what's going on in these icons with so many lines and all the subtle differences. I'd recommend basing these Flexbox "align" icons on our current align icons. Possibly naming them `align-flexbox-*` or even simply `align-*`

![align-horizontal-left](https://user-images.githubusercontent.com/1837101/95374735-f1865d80-089b-11eb-9287-3265b9ab6af8.png)

- [align-horizontal-center](https://materialdesignicons.com/icon/align-horizontal-center)
- [align-horizontal-left](https://materialdesignicons.com/icon/align-horizontal-left)
- [align-horizontal-right](https://materialdesignicons.com/icon/align-horizontal-right)
- [align-vertical-bottom](https://materialdesignicons.com/icon/align-vertical-bottom)
- [align-vertical-center](https://materialdesignicons.com/icon/align-vertical-center)
- [align-vertical-top](https://materialdesignicons.com/icon/align-vertical-top)

**Comment by zgover** [comment]
Yes, I actually didn't realize entirely how similar they are to the existing align icons initially. However to match the existing style it should be discussed how to display `space-between`, `space-around`. As far as the rest such as start, end and baseline could be substituted for the existing align top, bottom etc.

**Comment by darren-outdev** [comment]
Thanks for picking up this request! I do think there needs to be a rather 'non-developer' approach when thinking of designs - something that would make _kinda_ sense to someone who isn't a developer, and requires a very low level of thought for a person familiar with the technicality of controls... Hope that makes sense?

**Comment by SintijaPotaicuka** [comment]
Hi, 

I would like to pick up this issue.
I believe that since the issue was created things have changed and `space-between`, `space-around`, `stretch` has become more common in the designer community.

I would like to propose these icons that I am using for Style Bar UI:
<img width="616" alt="Screenshot 2022-03-30 at 11 24 30" src="https://user-images.githubusercontent.com/102287459/160799563-e33cc7e1-739a-4910-a841-1c51a40cef65.png">

With names:
<img width="1007" alt="Screenshot 2022-03-30 at 11 34 10" src="https://user-images.githubusercontent.com/102287459/160800525-439f91ab-b202-43c7-b735-b78ff36504df.png">

If I can get permission, I will create issues accordingly to the guidelines :)

**Comment by goyney** [comment]
You don't need permission, anyone is able to contribute! Go for it!

**Comment by zgover** [comment]
@SintijaPotaicuka @goyney @Templarian actually before you revel your time and effort, I have been noticing a more common pattern as the ⤵ below  picture and references, to which you are pretty close to already, but we should consider flexbox vs grid not only in available icons but also naming convention.

Flexbox ⤵ 
![image](https://user-images.githubusercontent.com/9044773/160841848-c04f3001-168d-4d34-a0d6-6e671be506c5.png)
_Source: [https://flexbox.malven.co/](https://flexbox.malven.co/)_

Grid ⤵ 
![image](https://user-images.githubusercontent.com/9044773/160842003-12412422-f292-4178-a3d1-8d13cf7e8d74.png)
_Source: [https://grid.malven.co/](https://grid.malven.co/)_

**Comment by goyney** [comment]
Quite honestly, Google has most of these already made and in Chrome and we should probably just include theirs and fill in any missing gaps in the same style.
![Screen Shot 2022-03-30 at 8 29 40 AM](https://user-images.githubusercontent.com/1015040/160846188-f93d7e4e-a95e-438f-b1fc-d88f926d64df.png)

**Comment by zgover** [comment]
That would be great if they didn't have such a unique approach with a heavy weighted approach and without substantially deviating for their pattern it was most likely producing ambiguous results lacking to little of change in visual appearance, and we'd probably realize it as well during our own attempt to finish what's missing and common.

<img width="307" alt="image" src="https://user-images.githubusercontent.com/9044773/160946683-a2e86ef5-f514-405d-909e-1a513cc2dec8.png">

Honestly, I think the easiest way would be to write an automation script create leaving a min of 2px spacing and 2–4px width of inactive element for items without attention, the min of 2-4px sized approach and write a script to dynamically render simple cases and generating its path rather than designing each on individually.

**Comment by SintijaPotaicuka** [comment]
My feeling is that google chrome icons do not fit in the style with MDI - they are using rounded while MDI seems to focus on sharp.

If someone can write a script to auto-generate, I would suggest using fewer rows/columns so they are easier to read as in the example - 2nd row. 
<img width="567" alt="Screenshot 2022-03-31 at 09 20 48" src="https://user-images.githubusercontent.com/102287459/161004364-b7f4c508-86fa-4acb-8bb0-bb64b2170452.png">
